src/util.py

```diff
@@ -296,3 +303,79 @@ def make_byr_cmap():
     warnings.filters.pop(0)
 
     return cmaps.BlueYellowRed
+
+
+def set_vector_density(ds, lat_density=1, lon_density=1, minDistance=0):
```

Comment:
Please move this function up around line 267. This is currently in a section with deprecated functions.